server/index.js

```diff
@@ -253,6 +256,7 @@ app.post(
     transformImageBody,
 	function(req, res, next) {
 		req.newFormat = true;
+        req.staging = staging;
```

Comment:
Could you fix the indentation on this

Reply:
My IDE is messed up on some dimension 
![image](https://user-images.githubusercontent.com/8428198/156849895-ae1504bd-7e07-48ec-8daa-a6f0b1761084.png)

Reply:
Fixed in outstanding PR https://github.com/roboflow-ai/inference-server/commit/287f26ada6a86658c568bd03af4fa53acc327839